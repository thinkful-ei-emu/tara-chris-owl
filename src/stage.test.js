import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './stage.js';
import renderer from 'react-test-renderer';

describe('Stage Component', () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<Stage participants={[  {
        id: 5,
        name: 'Ashla Attwool',
        avatar:
            'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1',
        inSession: true,
        onStage: true
    }]}/>, div);

        ReactDOM.unmountComponentAtNode(div);
  })

  it('renders the UI as expected', () => {
      const tree = renderer
        .create(<Stage participants={[  {
          id: 5,
          name: 'Ashla Attwool',
          avatar:
              'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1',
          inSession: true,
          onStage: true
      }]} />)
        .toJSON();
      expect(tree).toMatchSnapshot();  
      });
})