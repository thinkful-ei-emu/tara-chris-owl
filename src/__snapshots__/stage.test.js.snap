// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage Component renders the UI as expected 1`] = `
<section
  class="stage"
>
  <div
    className="stage-tile"
  >
    <h3>
      Ashla Attwool
    </h3>
    <button />
    <button />
    <img
      alt="Ashla Attwool's avatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
  </div>
</section>
`;
